content/docs/getting-started/futures.md

```diff
@@ -215,12 +210,9 @@ trait Stream {
     /// The type of the value yielded by the stream.
     type Item;
 
-    /// The type representing errors that occurred while processing the computation.
-    type Error;
-
     /// The function that will be repeatedly called to see if the stream has
     /// another value it can yield
-    fn poll(&mut self) -> Poll<Option<Self::Item>, Self::Error>;
+    fn poll_next(self: Pin<&mut Self>, cx: &mut Context) -> Poll<Option<Self::Item>>
```

Comment:
Nit: should this be 
```suggestion
    fn poll_next(self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Option<Self::Item>>
```
since the Rust 2018 idiom lints suggests that? 

I notice that the standard library definition of `Future::poll` does not use this syntax though, so maybe this should be the same?